packages/storage/src/persisted.ts

```diff
@@ -229,11 +229,11 @@ export const storageSync: PersistenceSyncAPI = [
 ];
 
 /**
- * postmessageSync - synchronize over post message API
+ * messageSync - synchronize over post message or broadcast channel API
  */
-export const postmessageSync: PersistenceSyncAPI = [
+export const messageSync = (channel: Window | BroadcastChannel = window): PersistenceSyncAPI => [
```

Comment:
Good job, that's quite elegant in my opinion

Reply:
Thanks!